osclib/stagingapi.py

```diff
@@ -827,6 +827,21 @@ def create_package_container(self, project, package, disable_build=False):
         :param disable_build: should the package be created with build
                               flag disabled
         """
+        if disable_build:
+            # keep build state in case builds is not disabled during the
+            # reqeust is superseding a new package
```

Comment:
I believe you are trying to say: `keep enabled build state for a superseding request providing a new package`